R/cache.R

```diff
@@ -236,15 +236,20 @@ cache <- function(uncached_function, key, salt, con, prefix = deparse(uncached_f
   ## Retain the same formal arguments as the base function.
   formals(cached_function) <- formals(uncached_function)
 
-  ## Check "force." name collision
-  if ("force." %in% names(formals(cached_function))) {
-    stop(sQuote("force."), " is a reserved argument in caching layer, ",
-         "collision with formals in the cached function.", call. = FALSE)
-  }
+  ## Check "force.", "dry." name collision
+  lapply(c("force.", "dry."), function(x) {
+    if (x %in% names(formals(cached_function))) {
+      stop(sQuote(x), " is a reserved argument in caching layer, ",
```

Comment:
can you rephrase that it's cachemeifyoucan, not a general caching layer? you never know how people can use open source software